php-practice/section1.php

```diff
@@ -132,6 +149,10 @@
 // PROBLEM 8: Print out $myArray from PROBLEM 6
 // =======================================================================
 
+$myArray = ["Bob Ross", 102, 3.14, true, "I am Russell and I eat arrays for breakfast"];
```

Comment:
I meant to imply that you don't need to re-create the array from the previous question. Since it's already defined. So let's remove this.